lib/plan-builder-generated.js

```diff
@@ -2296,7 +2296,7 @@ QName(...args) {
 /**
     * Returns a new sequence constructed from the value of target with the item at the position specified by the value of position removed.  Provides a client interface to a server function. See {@link http://docs.marklogic.com/fn.remove|fn.remove}
     * @method planBuilder.fn#remove
-    * @since 2.1.1
+     * @since 3.1.0
```

Comment:
Should these really have been updated? The "since" should not change unless the function is either new or has been modified.